app/services/search/google/search_service.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module Search
+  module Google
+    class SearchService < Search::BaseService
+      def initialize(keyword, language = 'en')
+        search_url = "https://www.google.com/search?q=#{keyword}&hl=#{language}&lr=#{language}"
```

Comment:
Never hard code such values. The static parts of the URL should be defined as constants.